### PR TITLE
fix: [FFM-5534]: Clean up of Getting Started Flow including Env Select

### PR DESCRIPTION
**Pull request by laura-mannus**
### Summary

A clean-up of the styling in the Getting Started flow. It is now more responsive as per ticket and matches the designs provided by Ksenia.

#### Screenshots

https://user-images.githubusercontent.com/103498167/205979182-00868acb-621d-4683-8c46-712c6d8d1c32.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cf/pages/onboarding/views/CreateAFlagView.tsx [hunk 1]
```diff
@@ -137,16 +137,18 @@ export const CreateAFlagView: React.FC<CreateAFlagViewProps> = ({ selectedFlag,
...
+          <Layout.Horizontal padding={{ top: 'medium', bottom: 'medium' }}>
+            <Spinner size={24} />
+          </Layout.Horizontal>
+        )}
+        {selectedFlag && (
+          <Layout.Horizontal margin={{ top: 'medium' }}>
+            <OnboardingSelectedFlag flagCreated={flagCreated} selectedFlag={selectedFlag} />
+          </Layout.Horizontal>
+        )}
+      </>
```
Comment (crystal-fong) [inline src/modules/75-cf/pages/onboarding/views/CreateAFlagView.tsx:151]: is this empty fragment needed if the top fragment wraps it and has no css

File: src/modules/75-cf/pages/onboarding/OnboardingDetailPage.tsx [hunk 2]
```diff
@@ -134,13 +134,13 @@ export const OnboardingDetailPage: React.FC = () => {
           }
         />
       </Layout.Horizontal>
-      <Container className={css.scrollOverflowY} height="100vh" padding={{ top: 'xxlarge', bottom: 'xxlarge' }}>
+      <Container className={css.scrollOverflowY} height="100%" padding={{ top: 'xxlarge', bottom: 'xlarge' }}>
```
Comment (crystal-fong) [inline src/modules/75-cf/pages/onboarding/OnboardingDetailPage.tsx:137]: could we maybe rename the css class? I don't think it classnames should be named based on the behaviour of the component and if the behaviour of that component changes we would have to rename it anyway

File: src/modules/75-cf/pages/onboarding/views/SelectEnvironmentView.tsx [hunk 3]
```diff
@@ -76,67 +76,70 @@ export const SelectEnvironmentView: React.FC<SelectEnvironmentViewProps> = ({
...
+        <Text font={{ variation: FontVariation.BODY }} color={Color.GREY_600} padding={{ bottom: 'medium' }}>
+          {getString('cf.onboarding.environmentDescription')}
+        </Text>
+        <Text font={{ variation: FontVariation.BODY }} color={Color.GREY_600} padding={{ bottom: 'xsmall' }}>
+          {getString('cf.onboarding.selectOrTypeEnv')}
+        </Text>
+      </Layout.Vertical>
+      {loading && <Icon name="spinner" size={16} color="blue500" />}
+      {!loading && (
+        <>
```
Comment (crystal-fong) [inline src/modules/75-cf/pages/onboarding/views/SelectEnvironmentView.tsx:93]: again here, is this fragment needed?

**Review by crystal-fong — APPROVED** [review]
just a couple of comments but no blockers 👍

File: src/modules/75-cf/pages/onboarding/views/CreateAFlagView.tsx (hunk 1 above)
Comment (laura-mannus) [reply src/modules/75-cf/pages/onboarding/views/CreateAFlagView.tsx:151]: really good spot thank you

File: src/modules/75-cf/pages/onboarding/OnboardingDetailPage.tsx (hunk 2 above)
Comment (laura-mannus) [reply src/modules/75-cf/pages/onboarding/OnboardingDetailPage.tsx:137]: Yeah totally agree have renamed

File: src/modules/75-cf/pages/onboarding/views/SelectEnvironmentView.tsx (hunk 3 above)
Comment (laura-mannus) [reply src/modules/75-cf/pages/onboarding/views/SelectEnvironmentView.tsx:93]: This guy is actually needed

**Review by Kira-Jennings — APPROVED** [review]
Looks good

**Review by crystal-fong — APPROVED** [review]

**Review by Kira-Jennings — APPROVED** [review]

**Pull request merged**